pkg/azurecluster/mutate_create.go

```diff
@@ -139,6 +140,20 @@ func (m *CreateMutator) Mutate(ctx context.Context, request *v1beta1.AdmissionRe
 		result = append(result, *patch)
 	}
 
+	azureClusterCR.Default()
+	{
+		var capiPatches []mutator.PatchOperation
+		capiPatches, err = patches.GeneratePatchesFrom(request.Object.Raw, azureClusterCR)
+		if err != nil {
+			return []mutator.PatchOperation{}, microerror.Mask(err)
+		}
+
+		capiPatches = patches.SkipPatchesForPath("/spec/networkSpec/vnet", capiPatches)
+		capiPatches = patches.SkipPatchesForPath("/spec/networkSpec/subnets", capiPatches)
```

Comment:
Ignoring these, as we set their configuration inside `azure-operator`, and we already default to a `control-plane` subnet in the code above.